front-end/src/App.js

```diff
@@ -37,6 +39,12 @@ const App = (props) => {
           <Route path="/">
             <Home />
           </Route>
+          <Route path="/Login">
```

Comment:
I'm pretty sure these routes need to be above the homepage's route since the Home route ("/") will get matched before ("/Login" or "/Signup"). I'd also change the path for these to be all lowercase, to fit with the rest of the path names